### DISCUSSION TITLE
components_to_compute

### DISCUSSION THREAD

**Discussion by AlvicePC** [ROBelgium/MSNoise#270] [I think there is a bug] [answered]
> Hello, im a beginer in msnoise usage. I follow the workflow but at the time to plot dvv this message appears in the console " No data for [ "ZZ" ] m5 f1" and open an empty figure- I dont have this problem with the other plots, searching in the doc i found that the folder ZZ response to the output of the parameter components_to_compute if it stay in default value, and i changed this to [ZZ,ZE,ZN,EZ,EE,EN,NZ,NE,NN] and i added another filter for reprocces data, but msnoise still only computing components ZZ and i still having the same message in console " No data for [ "ZZ" ] m5 f2" for the new filter.

**Comment by ThomasLecocq** [comment, marked as answer]
do you have the :

STACKS/005_DAYS/01/ 

folder?

does the msnoise plot ccftime -m5 work ?

**Comment by Amituofuo** [comment]
I am also a beginner with the same problem, Thomas Lecocq, and no data for {zz} m1 f1/No data for {zz} m5 f1 /No data for {zz} m10 f1 when plot dv/v.

**Reply by ThomasLecocq** [reply]
same question to you then ...
>do you have the :
>STACKS/005_DAYS/01/
>folder?
>does the msnoise plot ccftime -m5 work ?

**Reply by Amituofuo** [reply]
I tried to use test data to calculate, and finally MSnoise generated STACKS/005_ DAYS/01/folder. But the folder only includes YA_ UV05_ YA_ UV06 data, no my data. And I'm running step msnoise compute_ Cc, the calculation time is very short. I doubt that the data has not been calculated. Thank ThomasLecocq for helping me.

**Reply by ThomasLecocq** [reply]
well this is certainly not the way to do. Use your own data and config, the test config is making a lot of different changes to the config as it goes and it'll be difficult to adapt afterwards

**Reply by Amituofuo** [reply]
I see what you mean. When I recalculated according to my own settings and data, the STACKS/005DAYS/01/folder was not generated. The compute cc run time was very short. I doubt that CC was not calculated, so that the running time of the following steps was very short. Finally, there was no data. Can you help me point out which step might have problems?

**Reply by ThomasLecocq** [reply]
there is no point doing the next steps if CC doesn't work. Did you run "msnoise -v compute_cc" to see the debug messages ?

**Reply by Amituofuo** [reply]
I rerun msnoise - v compute cc and the result shows：
2022-09-28 07:06:41.383636 msnoise [pid 12564][DEBUG]: F.1339.Z Reading 1 Files
2022-09-28 07:06:41.384657 msnoise [pid 12564][DEBUG]: ERROR reading file G:/Win-Linux/16-19\1339.16-19.z.1624.SAC
2022-09-28 07:06:41.384657 msnoise [pid 12564][DEBUG]: F.1340.Z Reading 1 Files
2022-09-28 07:06:41.385631 msnoise [pid 12564][DEBUG]: ERROR reading file G:/Win-Linux/16-19\1340.16-18.Z.1624.SAC
2022-09-28 07:06:41.385631 msnoise [pid 12564][DEBUG]: Not enough data for this day !
2022-09-28 07:06:41.386919 msnoise [pid 12564][DEBUG]: Marking job Done and continuing with next !
At the same time, the following information is displayed in the Database:
F	1339	Z	G:/Win-Linux/16-19	1339.16-19.z.1624.SAC	2022-07-15 23:59:59	2022-07-16 23:59:59	86400.01	0.0	100.0	A
F	1340	Z	G:/Win-Linux/16-19	1340.16-18.Z.1624.SAC	2022-07-15 23:59:59	2022-07-17 00:00:00	86400.02	0.0	100.0	A

**Reply by ThomasLecocq** [reply]
well are your files readable ? is the disk mounted ? This error occurs for a single "read" command from obspy, so you can try reading the file again with obspy and check the traceback. Please open a new discussion as this is not related to the original question.
